File: Modules/Common/Techs/SIE.Web.Tech/Processs/Commands/SaveProcessCommand.js
SIE.defineCommand('SIE.Web.Tech.Processs.Commands.SaveProcessCommand', {
    extend: 'SIE.cmd.Save',
    meta: { text: "保存", group: "edit", iconCls: "icon-SaveEntity icon-blue" },
    doSave: function (view) {
        var me = this;
        var store = view.getData();
        store.data.each(function (p) {
            if (p.isEntityChildrenDirty()) {
                if (p._ParameterList && p._ParameterList.isDirty()) {
                    p._ParameterList.getData().items.forEach(function (p) { return p.dirty = true; });
                }
                if (p._CollectStepList && p._CollectStepList.isDirty()) {
                    p._CollectStepList.getData().items.forEach(function (p) { return p.dirty = true; });
                }
            }
        });
        me.callParent(arguments);
    },
});

File: Modules/Common/Techs/SIE.Web.Tech/Processs/Commands/SaveProcessCommand.cs
using SIE.Web.Command;

namespace SIE.Web.Tech.Processs.Commands
{
    [JsCommand("SIE.Web.Tech.Processs.Commands.SaveProcessCommand")]
    public class SaveProcessCommand : SaveCommand
    {
        /// <summary>
        /// 工序保存命令
        /// </summary>
        /// <param name="args">数据</param>
        /// <param name="scope">scope</param>
        /// <returns></returns>
        /// <remarks>在添加修改工序命令中调用</remarks>
        protected override object Excute(ViewArgs args, string scope)
        {
            base.Excute(args, scope);
            return true;
        }
    }
}
